autoload/neomake.vim

```diff
@@ -528,17 +530,16 @@ function! neomake#GetEnabledMakers(...) abort
     return enabled_makers
 endfunction
 
-function! s:HandleLoclistQflistDisplay(file_mode) abort
-    let open_val = get(g:, 'neomake_open_list')
-    if open_val
+function! s:HandleLoclistQflistDisplay(open_val, file_mode) abort
```

Comment:
The `defaults` from above are just the defaults for the makers - so it should work as before regarding this.

Speaking about it: having tests for this would prove that.. :)